app/components/mixins/auto-tracked.js

```diff
@@ -0,0 +1,25 @@
+import trackable from './trackable.js'
+
+export default {
+  mixins: [trackable],
+  mounted () {
+    this.$el.addEventListener('click', (event) => {
+      let element = event.target
+
+      while (element && element.nodeName !== 'A' && element.nodeName !== 'BUTTON') {
+        // find the closest link or button
+        element = element.parentNode
+      }
+
+      if (element) {
+        const eventName = `CTA ${element.innerText} clicked on ${window.location.pathname}`
```

Comment:
can we instead have the path of the link instead of text like Beta access as eventName?